app/routes/setup-page.tsx

```diff
@@ -28,10 +31,27 @@ function getStringFromFormData(
   // If the value was `null` or a `File` don't return anything
 }
 
-export async function clientLoader() {
+type ServerData = {
+  filesInBucket: string[];
+};
+
+/** The server side loader */
+export async function loader(): Promise<ServerData> {
+  return {
+    filesInBucket: await getSignedUrlsForItemsInBucket(),
```

Comment:
We probably don't really need/want this on the page anymore, but the collection of photos is making me laugh so I thought I'd leave them there for you 😆